build.gradle

```diff
@@ -61,6 +60,59 @@ sonar {
     }
 }
 
+def getFixAndFullVersion() {
+    def currentVersion = currentVersion.split("\\.")
+    def lastVersionNumber = currentVersion[currentVersion.length - 1]
+
+    return [lastVersionNumber, currentVersion]
+}
+
+def setVersionProperty(String newVersion) {
+    ant.propertyfile(file: "gradle.properties") {
+        entry(key: "currentVersion", value: newVersion)
+    }
+}
+
+tasks.register('removeSnapshotSuffix') {
+    doLast {
+        logger.info("Removing version suffix")
+        println("Current build version: $currentVersion")
+        def (lastVersionNumber, currentVersionParts) = getFixAndFullVersion()
+        lastVersionNumber = lastVersionNumber.replace('-SNAPSHOT', '')
+
+        def newVersion = currentVersionParts.take(2).join(".") + "." + lastVersionNumber
+        println("New build version: ${newVersion}")
+
+        setVersionProperty(newVersion)
+
+        exec {
```

Comment:
since we need to do this multiple times, could we create a small helper function to which we pass the name and value as arguments?